flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
I hadn't dared to test it but aarch64 actually works with a tiny patch, I'll remove this restriction.